src/story/story-quota.service.ts

```
@@ -0,0 +1,194 @@
+            userId,
+            currentMonth,
+            uniqueStoriesRead: 0,
+            bonusStories: 0,
+            lastBonusGrantedAt: now,
+          },
+        });
+        return usage;
+      }
+
+      // Calculate pending bonus stories to grant
+      const bonusesToGrant = this.calculatePendingBonuses(usage.lastBonusGrantedAt, now);
```

ESLint

[error] Replace `usage.lastBonusGrantedAt,·now` with `⏎········usage.lastBonusGrantedAt,⏎········now,⏎······`

(prettier/prettier)